src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,4 @@
-%h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
-}
+// %h1 {
```

Comment:
The %h1 placeholder is commented out, which might cause issues if it is being used in other parts of the project. If you no longer need it, consider removing it completely instead of commenting it out.